src/main/java/mate/academy/springboot/datajpa/model/Product.java

```diff
@@ -0,0 +1,26 @@
+package mate.academy.springboot.datajpa.model;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.FetchType;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.ManyToOne;
+import jakarta.persistence.Table;
+import java.math.BigDecimal;
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
+@Entity
+@Table(name = "products")
+public class Product {
```

Comment:
where are equals and hashcode and toString? do not forget that you have lazily initialized fields